polaris-icons/package.json

```diff
@@ -18,9 +18,10 @@
   },
   "scripts": {
     "build": "rollup -c",
+    "clean": "rm -rf .turbo node_modules dist",
     "dev": "rollup -c -w",
-    "test": "jest",
-    "clean": "rm -rf .turbo node_modules dist"
+    "optimize": "yarn svgo -f icons",
```

Comment:
Heads up @kyledurand and @samrose3, I don't think `yarn` is needed here. [`svgo`'s bin script](https://github.com/svg/svgo/blob/73f7002ab614554446e26acf1552c2505448b96f/package.json#L48) should automatically be symlinked to `node_modules/.bin` and thus available to the run-script:
See [here](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#bin):
> When this package is installed as a dependency in another package, the file will be linked where it will be available to that package either directly by npm exec or by name in other scripts when invoking them via npm run-script.

Reply:
Neat!